server/routerlicious/.changeset/ready-colts-bake.md

```diff
@@ -4,6 +4,10 @@
 
 Export ITimeoutContext, getGlobalTimeoutContext, and setGlobalTimeoutContext from @fluidframework/server-services-client
 
-The `@fluidframework/server-services-client` package now exports interface `ITimeoutContext` and functions `getGlobalTimeoutContext` and `setGlobalTimeoutContext`.
+The `@fluidframework/server-services-client` package now exports the following items. Please note they are not expected to be used outside of FluidFramework.
```

Comment:
@scottn12 Could you also decorate these APIs with the @internal release tag in their TSdocs in a follow-up PR? That won't do anything right now, but it'll set us up to hide those APIs from the package once Sonali's pipeline changes land.

Reply:
```suggestion
The `@fluidframework/server-services-client` package now exports the following items. These APIs are intended for internal use within the Fluid Framework only. They will be marked as internal APIs in a future release.
```